weatherAplication/Modules/Home/View/HomeViewController.swift

```diff
@@ -81,10 +96,41 @@ class HomeViewController: UIViewController {
 
 }
 
+extension HomeViewController: UITableViewDelegate, UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        let lastCity = viewModel.lastSearches
+        return lastCity.count
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "cell", for: indexPath as IndexPath)
+        cell.textLabel?.textColor = .yellow
+        cell.backgroundColor = .blue
```

Comment:
Zrób te cellki trochę ładniejsze, pomiędzy niebieskim a żółtym nie ma zbyt dużego kontrastu i nieprzyjemnie się to czyta. Jak używasz koloru tekstu to staraj się żeby jego kontrast z tłem był duży.